src/features/multi-step-form/utils/index.tsx

```
@@ -0,0 +1,102 @@
+  if (sns && sns.every((snsItem) => snsItem.link !== '')) {
+    formData.append('sns', JSON.stringify(sns)); // JSON.stringify로 배열 전체를 문자열로 변환
+  }
+
+  // 배열 필드: content
+  if (content && content.every((contentItem) => contentItem.link !== '')) {
+    formData.append('content', JSON.stringify(content)); // JSON.stringify로 배열 전체를 문자열로 변환
+  }
+
+  // 배열 필드: project
+  if (project && project.every((projectItem) => projectItem.link !== '')) {
+    formData.append('project', JSON.stringify(content)); // JSON.stringify로 배열 전체를 문자열로 변환
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)